lighthouse-core/audits/bootup-time.js

```diff
@@ -114,22 +114,28 @@ class BootupTime extends Audit {
    */
   static getExecutionTimingsByURL(trace) {
     const timelineModel = new DevtoolsTimelineModel(trace);
-    const bottomUpByName = timelineModel.bottomUpGroupBy('URL');
+    const bottomUpByURL = timelineModel.bottomUpGroupBy('URL');
     const result = new Map();
 
-    bottomUpByName.children.forEach((perUrlNode, url) => {
+    bottomUpByURL.children.forEach((perUrlNode, url) => {
       // when url is "" or about:blank, we skip it
       if (!url || url === 'about:blank') {
         return;
       }
 
-      const tasks = {};
+      const taskGroups = {};
       perUrlNode.children.forEach((perTaskPerUrlNode) => {
-        const taskGroup = WebInspector.TimelineUIUtils.eventStyle(perTaskPerUrlNode.event);
-        tasks[taskGroup.title] = tasks[taskGroup.title] || 0;
-        tasks[taskGroup.title] += Number((perTaskPerUrlNode.selfTime || 0).toFixed(1));
+        // eventStyle() returns a string like 'Evaluate Script'
+        const task = WebInspector.TimelineUIUtils.eventStyle(perTaskPerUrlNode.event);
+        // Resolve which taskGroup we're using
+        const groupName = taskToGroup[task.title] || group.other;
+        const groupTotal = taskGroups[groupName] || 0;
+        taskGroups[groupName] = groupTotal + (perTaskPerUrlNode.selfTime || 0);
       });
-      result.set(url, tasks);
+      Object.keys(taskGroups).forEach(groupName => {
+        taskGroups[groupName] = Math.round(taskGroups[groupName] * 10) / 10;
```

Comment:
Why do you want to reduce the number after the period?

Reply:
good call. nuked this.